packages/extension/src/providers/ethereum/libs/assets-handlers/blockscout.ts

```
@@ -0,0 +1,84 @@
+  const tokenBalancesUrl = `${NetworkEndpoints[chain]}api/v2/addresses/${address}/tokens?type=ERC-20`;
+
+  return Promise.all([
+    fetch(nativeTokenUrl).then((res) => res.json()),
+    fetch(tokenBalancesUrl).then((res) => res.json()),
+  ])
+    .then(([nativeResponse, tokenResponse]: [any, TokenResponse]) => {
+      if (!nativeResponse?.coin_balance || !tokenResponse?.items) {
+        return Promise.reject("Error fetching balance data");
+      }
+
+      if (isNaN(Number(nativeResponse.coin_balance))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)